gammapy/datasets/map.py

```diff
@@ -1360,6 +1311,79 @@ def reset_data_cache(self):
             if self.__dict__.pop(name, False):
                 log.info(f"Clearing {name} cache for dataset {self.name}")
 
+    def group_over_energy(self, e_edges=None, name=None):
```

Comment:
Minor: for consistency with the Map API I think I would prefer the name `resample_energy_axis()`...

Reply:
Done